docs/ide/reference/how-to-change-fonts-and-colors-in-the-editor.md

```diff
@@ -1,5 +1,5 @@
 ---
-title: Change editor fonts and colors
+title: Change editor fonts and colors in Visual Studio 2017
```

Comment:
"Visual Studio" is added as a title suffix globally, so we don't want to add it here. It ends up being duplicated in the title.